awx/ui_next/src/screens/Host/HostDetail/HostDetail.jsx

```diff
@@ -64,11 +162,25 @@ function HostDetail({ host, i18n }) {
             <Button
               aria-label={i18n._(t`edit`)}
               component={Link}
-              to={`/hosts/${id}/edit`}
+              to={
+                history.location.pathname.startsWith('/inventories')
+                  ? `/inventories/inventory/${match.params.id}/hosts/${match.params.hostId}/edit`
+                  : `/hosts/${id}/edit`
+              }
             >
               {i18n._(t`Edit`)}
             </Button>
           )}
+        {summary_fields.user_capabilities &&
+          summary_fields.user_capabilities.delete && (
+            <Button
```

Comment:
We could probably use the `< DeleteButton />` component here. I saw an example of it in the InventoryDetail view.